docs/src/pages.js

```diff
@@ -171,7 +171,14 @@ const pages = [
       },
     ],
   },
-  { title: 'Component API', ...findPages[0] },
```

Comment:
We need to keep this otherwise with miss all the other pages: 

<img width="244" alt="Capture d’écran 2020-09-16 à 15 34 48" src="https://user-images.githubusercontent.com/3165635/93344619-2c175000-f832-11ea-8605-70a0702070db.png">